cryptography/src/appleMain/kotlin/com/wire/kalium/cryptography/CoreCryptoCentralImpl.kt

```diff
@@ -61,19 +59,23 @@ private class Callbacks : CoreCryptoCallbacks {
 
 class CoreCryptoCentralImpl(
     private val cc: CoreCrypto,
-    private val rootDir: String,
-    private val defaultCipherSuite: UShort?
+    private val rootDir: String
 ) : CoreCryptoCentral {
 
-    override suspend fun mlsClient(clientId: CryptoQualifiedClientId): MLSClient {
+    override suspend fun mlsClient(
+        clientId: CryptoQualifiedClientId,
+        allowedCipherSuites: List<UShort>,
+        defaultCipherSuite: UShort
+    ): MLSClient {
         cc.mlsInit(MLSClientImpl.toUByteList(clientId.toString()))
         return MLSClientImpl(cc, defaultCipherSuite = defaultCipherSuite!!)
```

Comment:
shouldn't we remove this one as well?
```suggestion
        return MLSClientImpl(cc, defaultCipherSuite = defaultCipherSuite)
```